frontend/lib/parsing/README

```diff
@@ -1,22 +1,20 @@
 This directory contains the parser for Chapel which is implemented
 using the bison/flex tool set. The sources for the parser are
 
-chapel.lex
-chapel.ypp
+  chpl.lex # lex input file
+  chpl.ypp # bison input file
 
 and the generated files are
 
-flex-chapel.cpp,  ../include/flex-chapel.h
-bison-chapel.cpp, ../include/bison-chapel.h
+  flex-chpl-lib.h
+  flex-chpl-lib.cpp
+  bison-chpl-lib.h
+  bison-chpl-lib.cpp
 
 respectively.  The location for the generated files is
```

Comment:
With the output files listed across 4 lines but input files across 2 lines, I think "respectively" is no longer clear and should be removed.